apps/foundation-demo/src/app/misc/header/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="sidebar" [ngClass]="showSidebar ? 'sidebar-show' : 'sidebar-hide'">
+  <span class="sidebar__close" (click)="toggleShowSidebar()"><i class="fa fa-times"></i></span>
```

Comment:
Should we expose this button at a higher level?

Reply:
I don't think so. Right now, the responsibilities of the different sidebar components are:
`daff-sidebar` - manages classes that will show/hide the sidebar, and gives the sidebar base css classes
`sidebar-container` - manages sidebar redux state
`sidebar-view` - passes sidebar redux state to sidebar component
`sidebar` - dispatches any changes to state from the user and relays show/hide status to `daff-sidebar` from state.

I could move this button to `sidebar-view` and transclude it into `sidebar`, but then `sidebar-view` would share some of `sidebar`'s responsibility of dispatching changes to state.